modules/osdk-hh-create-helm-api.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/helm/osdk-hybrid-helm.adoc
+
+:_content-type: PROCEDURE
+[id="osdk-hh-create-helm-api_{context}"]
+= Creating a Helm API
+
+Use the Operator SDK CLI to create a Helm API.
+
+.Procedure
+
+* Run the following command to create a Helm API with group `cache`, version `v1alpha1`, and kind `Memcached`:
++
+[source,terminal]
+----
+$ operator-sdk create api \
+    --plugins helm.sdk.operatorframework.io/v1 \
+    --group cache \
+    --version v1alpha1 \
+    --kind Memcached
+----
+
+[NOTE]
+====
+This procedure configures the Operator project to watch the `Memcached` resource with API version `v1alpha1` and also scaffolds a boilerplate Helm chart. Instead of creating the project from the boilerplate Helm chart scaffolded by the Operator SDK, you can also use an existing chart from your local file system or remote chart repository.
```

Comment:
This procedure creates the API, right? Are these additional tasks? _This procedure *also* configures..._

And, where this is a separate module, should you use the full name of the Operator?

Reply:
Cleaned this up.